### ISSUE TITLE
Issue with WebRTC example - Delegate 'Func<WebRTCWebSocketPeer>' does not take 1 arguments

### ISSUE THREAD

**Issue by W3AXL** [sipsorcery-org/sipsorcery#1033]
I might just be doing something wrong, but I'm trying to get the basic WebRTC example up and running and hitting an error when defining the `WebRTCWebSocketPeer` websocket service and trying to assign the `CreatePeerConnection` function:

```csharp
Wss = new WebSocketServer(address, port);
Wss.AddWebSocketService<WebRTCWebSocketPeer>("/", (peer) => peer.CreatePeerConnection = CreatePeerConnection);
```
`Error: Delegate 'Func<WebRTCWebSocketPeer>' does not take 1 arguments`

Am I missing something or has the instantiation for the WebRTC peer changed since the docs were last updated? This is on a .NET 6 project in VS 2022.

**Comment by W3AXL** [comment]
I believe the problem was caused by using [sta's WebSocketSharp](https://github.com/sta/websocket-sharp) as opposed to the sipsorcery version

**Comment by W3AXL** [comment]
Any assistance? This seems to be a pretty major and breaking bug. I've tried .NET6 and .NET8 with the same result - there's no way to properly instantiate the `WebRTCWebSocketPeer`.

**Comment by sipsorcery** [comment]
Will take a look.

**Comment by W3AXL** [comment]
I think I figured out the issue.

Originally, I was using the `sta/websocket-sharp` package and that resulted in the error above. I then tried switching to the `sipsorcery/websocket-sharp` package but the error did not resolve.

However, this morning I removed and reinstalled all packages in my solution and it seems to have fixed the issue. I'm not sure if it was a package cache issue or what, but nothing else seemed to resolve it (including switching around to a bunch of different .NET versions).

It should probably be noted somewhere in the docs (and apologies if it is and I missed it) that you must use the sipsorcery version of websocket-sharp in order for things to work properly.
